### PR TITLE
Add Kubelet metrics collect to Kubernetes integration

### LINKED ISSUE #112: Add new kubernetes/kubernetes-monitoring integration

### PR DESCRIPTION
**Pull request by calebhailey**
Closes #112

### CODE REVIEW

**Comment by calebhailey** [comment]
This needs to be updated to use the new `http-get` plugin. I'll do that now.

**Comment by jspaleta** [comment]
Need to finish replacing `wget` with `http-get` everywhere now?  
`http-get` is used in sensu-resources check command references but `wget` is used in the sensu-integration  check command patches

**Review by jspaleta — CHANGES_REQUESTED** [review]
`wget` usage in sensu-integration needs to be replaced with `http-get` usage to match sensu-resources.

File: integrations/kubernetes/kubernetes-monitoring/sensu-integration.yaml [hunk 1]
```diff
@@ -56,15 +126,88 @@ spec:
...
+            name: "[[incidents_pipeline]]"
+    - resource:
+        api_version: core/v2
+        type: CheckConfig
+        name: kubelet-metrics
+      patches:
+        - op: replace
+          path: /spec/command
+          value: >-
+            wget -q -O-
```
Comment (jspaleta) [inline integrations/kubernetes/kubernetes-monitoring/sensu-integration.yaml:159]:
```suggestion
            http-get
```

**Review by jspaleta — APPROVED** [review]

**Pull request merged**